.travis.yml

```diff
@@ -10,14 +10,11 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
-before_install:
-  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
```

Comment:
The build succeeds without this - https://travis-ci.org/aspnet/Razor/builds/101934136. Why do we need it in this repo in particular and not others?